src/docs/reference/modules/AutoSetup/README.md

```diff
@@ -70,9 +70,26 @@ Auto-Setup parameters are defined in appsettings.json. Example excerpt:
     `/autosetup` - trigger installation of the Root tenant.
     `/mytenant/autosetup` - auto-install mytenant.
 
-### Environment Variables
+### User Secrets and Environment Variables
+
+Since JSON configuration contains admin-sensitive information, it is recommended to use user secrets or environment variables instead.
+
+[User secrets](https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets#secret-manager) are available during local development, and they are stored as JSON files. This means you can move the whole configuration from appsettings.json as-is. Alternatively, you can set each option directly from the command line (this will flatten any existing structures in the secrets.json file):
```

Comment:
```suggestion
[User secrets](https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets#secret-manager) are available during local development, and they are stored as JSON files. This means you can move the whole configuration from `appsettings.json` as-is. Alternatively, you can set each option directly from the command line (this will flatten any existing structures in the `secrets.json` file):
```